docusaurus.config.js

```diff
@@ -77,6 +77,45 @@ module.exports = {
         {
           type: "docsVersionDropdown",
           position: "left",
+          dropdownItemsAfter: [
+            {
+              type: "html",
+              value: '<hr class="dropdown-separator">',
+            },
+            {
+              type: "html",
+              className: "dropdown-archived-versions",
+              value: "<b>Archived versions</b>",
+            },
+            // ["1.3", "1.2", "1.1", "1.0", "0.26", "0.25"].map(
+            // ([versionName]) => ({
+            {
+              label: "1.3",
+              href: "https://unsupported.docs.camunda.io/1.3/",
```

Comment:
These URLs have been updated recently -- we'll put all archived versions in the same subdomain, but each one will have its own path on that server.